pkg/cli/cluster_migrate_multinode_storage.go

```diff
@@ -147,7 +147,7 @@ func continueWithStorageMigration() bool {
 	fmt.Print("Would you like to continue with the migration now? (Y/n) ")
 	var answer string
 	fmt.Scanln(&answer)
-	return answer == "" || strings.ToLower(answer) == "y"
+	return strings.ToLower(answer) == "y"
```

Comment:
i believe you need to change this to `(y/N)` if you want to default to no

Reply:
done